index.js

```diff
@@ -1,32 +1,52 @@
-const client = require('./lib/statsd');
+/*
+ * Initialize a hot-shots client.
+ */
 
-const isNotEmpty = (obj) => obj && Object.getOwnPropertyNames(obj).length;
+const os = require('os');
 
-function keyValuesToString(keyValues) {
-  return Object.keys(keyValues)
-    .reduce((r, k) => r.concat(`${k}=${keyValues[k]}`), [])
-    .join(',');
+const statsdClient = require('hot-shots');
+const strigoLogger = require('strigo-node-logger');
```

Comment:
I don't think it's a good idea to introduce here the dependency for a logger, anyway each service will create a logger for itself with its specific setup.
So, I'd suggest or:
1. Drop the logs here - how much value do they add here?
2. Pass logger as prop to the setup function
3. Change implementation to use some wrapper class extending EventEmitter and fire events on initialization with the message for the logger and on fail close - whatever events there will be. This, however, makes it more complicated and not sure that it's worth it.